web3/providers/websocket.py

```diff
@@ -55,13 +56,13 @@ class WebsocketProvider(JSONBaseProvider):
     logger = logging.getLogger("web3.providers.WebsocketProvider")
     _loop = None
 
-    def __init__(self, endpoint_uri=None):
+    def __init__(self, endpoint_uri=None, **kwargs):
```

Comment:
I'm not wild about letting the `websockets` connection parameters dictate the whole keyword argument namespace here. How about adding a single dict variable, like `connection_params={'max_size': 1}`?

That way there is no chance of a future namespace conflict or confusing mismatch. (we already have one mismatch: a value is called `uri` in websockets, but `endpoint_uri` here).

Reply:
I chose the name `websocket_kwargs` instead, to be consistent with `request_kwargs` in `HttpProvider`.